commands/env.cmd

```diff
@@ -18,6 +18,42 @@ DOCKER_COMPOSE_ARGS=()
 DOCKER_COMPOSE_ARGS+=("-f")
 DOCKER_COMPOSE_ARGS+=("${WARDEN_DIR}/environments/${WARDEN_ENV_TYPE}.base.yml")
 
```

Comment:
I think would be good to name file as `mariadb`, not as `db`, so there will be ability to use MySQL instead